From c6966e016ddf86cfff5c505f79b849959c0a5227 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Thu, 18 Jul 2024 15:58:00 -0700
Subject: [PATCH] net/ipv4/Kconfig: Convert goo.gl URL to the original one

Google URL shortner will be unavailable soon[1].  Convert a shortened
URL to original one.

[1] https://developers.googleblog.com/en/google-url-shortener-links-will-no-longer-be-available/

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 net/ipv4/Kconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/net/ipv4/Kconfig b/net/ipv4/Kconfig
index 8e94ed7c56a0..6d2c97f8e9ef 100644
--- a/net/ipv4/Kconfig
+++ b/net/ipv4/Kconfig
@@ -661,7 +661,8 @@ config TCP_CONG_CDG
 
 	  For further details see:
 	    D.A. Hayes and G. Armitage. "Revisiting TCP congestion control using
-	    delay gradients." In Networking 2011. Preprint: http://goo.gl/No3vdg
+	    delay gradients." In Networking 2011. Preprint:
+	    http://caia.swin.edu.au/cv/dahayes/content/networking2011-cdg-preprint.pdf
 
 config TCP_CONG_BBR
 	tristate "BBR TCP"
-- 
2.39.2

